docs/conversions.md

```diff
@@ -1,8 +1,171 @@
 # @title Rule Conversions
 
-## Conversion Examples
+# Conversion Examples
 
-### DSL
+## JSScripting
+
+```java
+var email = "juliet@capulet.org"
+
+rules.JSRule({
+  name: "Balcony Lights ON at 5pm",
+  description: "Light will turn on when it's 5:00pm",
+  triggers: [triggers.GenericCronTrigger("0 0 17 * * ?")],
+  execute: (event) => {
+    items.getItem("BalconyLights").sendCommand("ON");
+    actions.NotificationAction.sendNotification(email, "Balcony lights are ON");
+  },
+  tags: ["Balcony", "Lights"],
+  id: "BalconyLightsOn"
+});
+```
+
+Ruby
+
+```ruby
+EMAIL = "juliet@capulet.org"
+
+rule "Balcony Lights ON at 5pm" do
+  description "Light will turn on when it's 5:00pm"
+  every :day, at: '5pm' # This can be written as: `cron "0 0 17 * * ?"` if preferred
+  tags: %w[Balcony Lights]
+  uid: "BalconyLightsOn"
+  run do |event|
+    BalconyLights.on
+    NotificationAction.send_notification(EMAIL, "Balcony lights are ON")
+  end
+end
+```
+
+Terse rules
+
+JRubyScripting supports {OpenHAB::DSL::Rules::Terse Terse syntax} as well as the normal,
+more traditional syntax. Some attributes such as rule description and tags cannot be set
+using the Terse syntax. They can be specified using the full rule syntax.
+
+---
+JS:
+
+```java
+// Basic rule, when the BedroomLight1 is changed, run a custom function
+rules.when().item('BedroomLight1').changed().then(e => {
+    console.log("BedroomLight1 state", e.newState)
+}).build();
+```
+
+Ruby:
+
+```ruby
+# Basic rule, when the BedroomLight1 is changed, run a custom function
+changed(BedromLight1) { |event| logger.info "#{event.item.name} state #{event.state}" }
+```
+
+---
+JS:
+
+```java
+// Turn on the kitchen light at SUNSET
+rules.when().timeOfDay("SUNSET").then().sendOn().toItem("KitchenLight").build("Sunset Rule","turn on the kitchen light at SUNSET");
+```
+
+Ruby:
+
+```ruby
+# Turn on the kitchen light at SUNSET
+channel("astro:sun:home:set#event", name: "Sunset Rule") { KitchenLight.on }
+```
+
+---
+JS:
+
+```java
+// Turn off the kitchen light at 9PM and tag rule
+rules.when().cron("0 0 21 * * ?").then().sendOff().toItem("KitchenLight").build("9PM Rule", "turn off the kitchen light at 9PM", ["Tag1", "Tag2"]);
+```
+
+Ruby:
+
+```ruby
+# Turn off the kitchen light at 9PM and tag rule
+every(:day, at: '9pm', name: "9PM Rule") { KitchenLight.off }
+```
+
+---
+JS:
+
+```java
+// Set the colour of the hall light to pink at 9PM, tag rule and use a custom ID
+rules.when().cron("0 0 21 * * ?").then().send("300,100,100").toItem("HallLight").build("Pink Rule", "set the colour of the hall light to pink at 9PM", ["Tag1", "Tag2"], "MyCustomID");
+```
+
+Ruby:
+
+```ruby
+# Set the colour of the hall light to pink at 9PM, tag rule and use a custom ID
+every(:day, at: '9pm', name: "Pink Rule", id: "MyCustomID") { HallLight.command("300,100,100") }
```

Comment:
any reason to prefer `command` over `<<`?
maybe show them both?

Reply:
I primarily didn't want to scare them off. Wanted to show them that it's somewhat familiar... but nevertheless, I've changed it to `<<`